02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -0,0 +1,537 @@
+| Elm to PureScript Cheatsheet | Reference | Elm developers considering PureScript | GitHub Repo
+| Differences of PureScript from Elm | Reference | Elm developers consider PureScript (or vice versa) | GitHub Gist
+| Documentation Repo's "Differences from Haskell" | Reference | Haskell developers considering PureScript | Markdown file
+
+Here's what has been added to the above since then:
+
+| Name | Type | Audience | Medium |
+| -- | -- | -- | -- |
+| Real World App | How To Guide? Reference? | PureScript developers | Code Example
+| MultiPac | Reference? | PureScript developers | Code Example
+| PureScript: Jordan's Reference | Hook + Getting Started + How to + Explanation + Reference | New Learners | GitHub Repo
+| A Guide to the PureScript Numeric Hierarchy | Explanation | New Learners | Read the Docs
+| Make the Leap from JavaScript to PureScript | Getting Started + Explanation | Javascript developers | Blog Post series
+
+## Why is PureScript's Documentation Lacking and How Do We Improve It?
+
+Here's the short version. Each paragraph will have a link that explains its summary in more detail.
+
+PureScript is not _currently_ trying to be the next "mainstream language." Rather, it values "power" over "popularity" and so works towards those ends. [further explained here]()
+
+While developing the language, core contributors incurred a lot of responsibility (e.g. library maintenance, documentation updates, updating Pursuit and other related sites, etc.). Over time, they became spread thin and one even burned out. [further explained here]()
+
+Thus, they cannot quickly respond to most people's questions or contributions. With the small amount of time they do have, they are focusing on adding new language features, not improving the documentation situation. The language's development is steady (6-week release cycle) but slow. [further explained here]()
+
+However, it seems they are (wisely) postponing "breaking changes" until all can be done at once. Otherwise, they will ~spend~ waste even more of their time updating the ecosystem to account for the breaking changes. There is also some disagreement about when those changes should be done. [further explained here]()
+
+Due to the looming "breaking changes," those who want to improve the documentation situation are often discouraged from improving it. They tend to think, "Why invest many hours into something that will be outdated in a few months? Why not invest those hours into something else, like an interesting project or learning something new?" [further explained here]()
+
+The following statement is not to blame people or point fingers. Rather, it summarizes our current reality:
+> Core contributors are doing great and their pace is just fine. Still, we're in a sort of catch-22 situation. The longer the core contributors take to implement these breaking changes, the longer most documentation writers will wait to avoid unnecessary updates, and the longer we'll be in this situation.
```

Comment:
I think the current rendering of this is actually offensive and hypocritical. It comes off as "We aren't going to blame the problem on anyone. `*immediately points a finger at core contributors as source of problem*`"

It might be better to include a summary of the following links, as those are the things that people would likely immediately ask afterwards:
- Why not delegate?
- Why not fund developers?

Such a rendering might be
> Fixing this situation is not as simple as it sounds. You can't "just delegate" their work to anyone. Others attempted to fund core contributors. However, the effort was likely stopped due to not having a clear idea of how the funds would be spent. [further explained here]()

Reply:
I've replaced the original with the above rendering in the latest commit.